Formula/rakudo.rb

```diff
@@ -0,0 +1,24 @@
+class Rakudo < Formula
+  desc "Production-ready, stable implementation of the Perl 6 language"
```

Comment:
`s/Production-ready, stable i/I/`

Reply:
If you want title case here, shouldn't we `s/stable/Stable`? Capitalizing `implementation` seems inconsistent.

Reply:
`desc "Implementation of the Perl 6 language"`

Reply:
Oh, in that case: the description as is was lifted from the rakudo.org homepage. I disagree with removing the "Production-ready, stable", as doing so insinuates there are other implementations of Perl 6, which there are not. It would be like saying CPython is "Implementation of the Python language".

Reply:
We try to avoid laudative terms (“excellent”, etc.) in descriptions, to focus on facts. How about `Perl 6 compiler targeting MoarVM` (which is the first sentence of the Wikipedia page)?

`rakudo-star` has `Perl 6 compiler`, and given it is the _default_ download from the Rakudo project, it should remain as such.